src/plugins/intel_gpu/tests/unit/test_cases/convolution_gpu_test.cpp

```diff
@@ -12064,3 +12063,99 @@ TEST(convolution_f16_fw_gpu, basic_bfzyx_b_fs_zyx_fsv16) {
         ASSERT_NEAR(reference_output[i], output_mem_lock[i], 1.0);
     }
 }
+
+TEST(conv_dyn_test, changed_batch_convolution_test_reorder_cache_mismatch) {
+    auto& engine = get_test_engine();
+
+    if (!engine.get_device_info().supports_immad)
+        return;
+
+    auto in_shape = ov::Shape{25,3,32,32};
+    auto new_shape = ov::Shape{10,3,32,32};
+    auto weight_shape = ov::Shape{1152,3,14,14};
```

Comment:
nit: this tensor size seems quite large. I'm afraid this would take long time to run and validate. What about using smaller shape, maybe something like `256` instead of `1152`?

Reply:
Applied